gap/ClassicalMaximals.gi

```diff
@@ -827,3 +827,17 @@ function(n, q, classes...)
 
     return maximalSubgroups;
 end);
+
+
+BindGlobal("C1SubgroupsSymplecticGroupGeneric",
+function(n, q)
+    local listOfks, result;
+    listOfks := [1..QuoInt(n, 2) - 1];
```

Comment:
This is not covered, but my understanding this is because this function will only be called in a future PR, correct?

Reply:
Correct. Theses PRs are only for the individual Classes, once they are all done we will call the generic functions from a function along the lines of `MaximalSubgroupClassRepsSymplecticGroup`. The generic functions will only be called from there.